### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by HaritzPuerto**
# What does this PR do?
Updating the docs

## Before submitting / marking as 'ready to review'
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you make sure to update the documentation with your changes?

### CODE REVIEW

**Comment by HaritzPuerto** [comment]
I believe this should work, but I can't check this in my local. I think once the testing VM is ready, we can easily check it there and merge it

File: docs/home/overview/architecture.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    - Fine-tuned HuggingFace (HF) Transformer models for QA.
+    - The backbone Transformer models for Adapters (e.g., bert-base-uncased)
+- **Skills**: module defines a QA pipeline. The predefined available pipelines are: 
+  - **Extractive**: the input is a question and optionally a context (i.e., a short passage) and the answer is a subspan of the context. If the context is not given, the Skill needs to make use of a datastore and an index to retrieve a relevant document that may contain the answer.
+  - **Multiple Choice**: the input is a question, a list of candidate answers, and optinally a context. The Skill has to identify the correct answer from the list of candidate answers.
+  - **Categorical**: it is a special case of multiple-choice where only two candiate answers are available, true/false or yes/no.
+  - **Abstractive**: similar to extractive, the input is a question and a context, but the answer is not a subspan of the context, instead it is a free-form text based on the context.
+  - **Information Retrieval (IR)**: the input is a question and the Skill needs to retrieve a relevant document.
+  - **User Interface**: allows the user to interact with the Skills.
```
Comment (timbmg) [inline docs/home/overview/architecture.md:24]: Seems like this is indented but shouldnt?

Comment (HaritzPuerto) [reply docs/home/overview/architecture.md:24]: It should be indented. It should look like this:
image

Comment (timbmg) [reply docs/home/overview/architecture.md:24]: right but the UI is one to many no?

Comment (HaritzPuerto) [reply docs/home/overview/architecture.md]: oh good catch! I didn't see it!. Fixed, thanks!

**Pull request merged**